test/core/simd/simd_const.wast

```diff
@@ -1076,7 +1073,7 @@
   (func (export "i32x4.test") (result v128) (return (v128.const i32x4 0x0bAdD00D 0x0bAdD00D 0x0bAdD00D 0x0bAdD00D)))
   (func (export "i32x4.smax") (result v128) (return (v128.const i32x4 0x7fffffff 0x7fffffff 0x7fffffff 0x7fffffff)))
   (func (export "i32x4.neg_smax") (result v128) (return (v128.const i32x4 -0x7fffffff -0x7fffffff -0x7fffffff -0x7fffffff)))
-  (func (export "i32x4.inc_smin") (result v128) (return (i32x4.add (v128.const i32x4 -0x80000000 -0x80000000 -0x80000000 -0x80000000) (v128.const i32x4 1 1 1 1))))
+  (; (func (export "i32x4.inc_smin") (result v128) (return (i32x4.add (v128.const i32x4 -0x80000000 -0x80000000 -0x80000000 -0x80000000) (v128.const i32x4 1 1 1 1)))) ;)
```

Comment:
I commented this out because it uses i32x4.add and I did not want to implement i32x4.add inside this patch. In future patches when we have i32x4.add implemented, we can uncomment this.
Actually, now that I think of it, it doesn't really need to be commented out, since we are not running these tests by default yet. I will upload a patch to uncomment this.

Reply:
Fixed in #159 159